README.md

```diff
@@ -2,7 +2,84 @@
+
+Early contributors include developers from GeoPandas, GeoTrellis, OpenLayers, Vis.gl, Voltron Data, Microsoft, Carto, Azavea, Planet & Unfolded. 
+Anyone is welcome to join us, by building implementations, trying it out, giving feedback through issues and contributing to the spec via pull requests.
+
+Note that version 0.1 only supports 2d geometries, as we are not supporting [EWKB](https://libgeos.org/specifications/wkb/#extended-wkb) or 
+[ISO WKB](https://libgeos.org/specifications/wkb/#iso-wkb). We expect to expand WKB to support 3D in the future, if you have thoughts please 
+sound in on #18 and/or follow along there.
+
+## Goals
+
+There are a few core goals driving the initial development.
+
+* **Establish a great geospatial format for workflows that excel with columnar data** - Most data science and 'business intelligence' workflows have been moving
+ towards columnar data, but current geospatial formats can not be as efficiently loaded as other data. So we aim to bring geospatial data best practices to one
+ of the most popular formats, and hopefully establish a good pattern for how to do so.
+* **Introduce columnar data formats to the geospatial world** - And most of the geospatial world is not yet benefitting from all the breakthroughs in data analysis
+ in the broader IT world, so we are excited to enable interesting geospatial analysis with a wider range of tools.
+* **Enable interoperability among cloud data warehouses** - BigQuery, Snowflake, Redshift and others all support spatial operations but importing and exporting data 
+ with existing formats can be problematic. All support and often recommend Parquet, so defining a solid GeoParquet can help enable interoperability.
+* **Persist geospatial data from Apache Arrow** - GeoParquet is developed in parallel with a [GeoArrow spec](https://github.com/geopandas/geo-arrow-spec), to 
+ enable cross-language in-memory analytics of geospatial information with Arrow. Parquet is already well-supported by Arrow as the key on disk persistance format.
+
+And our broader goal is to innovate with 'cloud-native vector' providing a stable base to try out new ideas for cloud-native & streaming workflows. 
+
+
+## Features
+
+A quick overview of what geoparquet supports (or at least plans to support).
+
+* **Multiple spatial reference systems** - Many tools will use GeoParquet for high-performance analysis, so it's important to be able to use data in its
+ native projection. But we do provide a clear default recommendation to better enable interoperability, giving a clear target for implementations that don't want to
+ worry about projections.
+* **Multiple geometry columns** - There is a default geometry column, but additional geometry columns can be included.
+* **Great compression / small files** - Parquet is well-designed to compress very well, so data benefits by taking up less disk space & being more efficient over
```

Comment:
```suggestion
* **Great compression / small files** - Parquet is designed to compress very well, so data benefits by taking up less disk space & being more efficient over
```

(to avoid twice "well" in the same sentence, just a language suggestion, I am no native speaker so defer to your judgement :))

Reply:
Your judgement is correct :) - I'm a lazy native-writer, and very much appreciate your fixes.